src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -229,22 +230,26 @@ public static void setFormatsToRepoConfigs(List<RepoConfiguration> configs, List
     }
 
     /**
-     * Sets each {@code RepoConfiguration} in {@code configs} to ignore its standalone config, if
+     * Sets each {@link RepoConfiguration} in {@code configs} to ignore its standalone config, if
      * {@code ignoreAllStandaloneConfigs} is true.
      */
-    public static void setStandaloneConfigIgnoredToRepoConfigs(
-            List<RepoConfiguration> configs, boolean ignoreAllStandaloneConfigs) {
+    public static void setStandaloneConfigIgnoredToRepoConfigs(List<RepoConfiguration> configs,
+            boolean ignoreAllStandaloneConfigs) {
         if (ignoreAllStandaloneConfigs) {
             configs.forEach(config -> config.setStandaloneConfigIgnored(true));
         }
     }
 
+    /**
+     * Checks if there are any repos in {@code configs} for which RepoSense is trying to blame line changes
+     * caused by commits in the ignore commit list to any available previous authors who changed those lines.
+     */
```

Comment:
I personally don't think this method requires a Javadocs. I think it falls under the self-explanatory category where we're checking if a specific boolean flag is ticked. 

If necessary, I think something like
```suggestion
    /**
     * Checks if any of the {@code configs} is finding previous authors for commit analysis.
     */
```